### DISCUSSION TITLE
libvips.so.42: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by rajkumarks7** [libvips/libvips#2053] [Q&A]
**Describe the bug**
i am using the bimg getting this error how to solve it.
./workahub: "error while loading shared libraries: libvips.so.42: cannot open shared object file: No such file or directory"
when i run the exe from the terminal in Ubuntu i am getting the above error 

**Environment**
(please complete the following information)
 - OS: Ubuntu 18.04
 - Vips: 8.10 
 - Go:1.14

**Comment by lovell** [comment]
Hi, please can you provide more information about how you've installed libvips.

Did you see https://github.com/h2non/bimg#installation ?

What does "when i run the exe" mean?

**Comment by rajkumarks7** [comment]
like i followed the whole documentaion of the libvips https://libvips.github.io/libvips/install.html

i built a exe or binary for my program which use the bimg for processing the thumbnails

**Comment by rajkumarks7** [comment]
> Hi, please can you provide more information about how you've installed libvips.
> 
> Did you see https://github.com/h2non/bimg#installation ?
> 
> What does "when i run the exe" mean?
actually i developed a desktop application i am using bimg for the processing of the image

yes i used below link for installation
https://libvips.github.io/libvips/install.html

**Comment by giovannibonetti** [comment]
~I'm having the same problem, but I can share my setup more easily since we use Docker here~

Nevermind, we had the same problem, but only on our Review apps running on Heroku. I'll try the steps described at https://github.com/libvips/ruby-vips/issues/219 and see if they solve the problem.
